eng/common/scripts/add-issue-assignees.ps1

```diff
@@ -0,0 +1,55 @@
+param(
+    [Parameter(Mandatory = $true)]
+    $RepoOwner,
+
+    [Parameter(Mandatory = $true)]
+    $RepoName,
+
+    [Parameter(Mandatory = $false)]
+    $GitHubUsers = "",
+
+    [Parameter(Mandatory = $true)]
+    $IssueNumber,
+  
+    [Parameter(Mandatory = $true)]
+    $AuthToken
+)
+
+function AddMembers($memberName, $additionSet) {
```

Comment:
This function looks very much the same as the on in the add-pullrequest-reviewers.ps1 we should be sharing and potentially combining the scripts.

Reply:
I moved it all into the submit-pullrequest scr